src/main/java/mate/academy/spring/model/User.java

```diff
@@ -14,7 +16,10 @@ public class User {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     @Column(unique = true)
+    @NotNull
```

Comment:
To use validation as far as I understood we need to place that annotations in dto classes, still validation is not demanded in our task